app/components/ak-loader/index.hbs

```diff
@@ -1,39 +1,60 @@
+{{!@glint-nocheck: or, not, and}}
 <span
   data-test-ak-loader
   ...attributes
   {{style width=(concat this.size 'px') height=(concat this.size 'px')}}
-  class='ak-loader-root'
   local-class='ak-loader-root'
   role='progressbar'
 >
+  <span local-class='ak-loader-svg-label'>
```

Comment:
can we just have the label block instead this would the keep the component lot more simpler

Reply:
It needs to be positioned absolutely. So this wrapping element is necessary.

Reply:
has-block check can be above wrapping element and label block inside the wrapper